binding.gyp

```diff
@@ -22,7 +22,21 @@
       ["OS == 'mac'", {
         "cflags+": ["-fvisibility=hidden"],
         "xcode_settings": {
-          "GCC_SYMBOLS_PRIVATE_EXTERN": "YES" # -fvisibility=hidden
+          # -fvisibility=hidden
+          "GCC_SYMBOLS_PRIVATE_EXTERN": "YES",
+
+          # Set minimum target version because we're building on newer
+          "MACOSX_DEPLOYMENT_TARGET": "10.10",
```

Comment:
I don't know if this actually works, or what our effective minimum version is (or was) without it.

Reply:
Should be [10.7](https://github.com/nodejs/node/blob/v10.0.0/common.gypi#L464) for Node.js 10.x, and [10.10](https://github.com/nodejs/node/blob/v12.0.0/common.gypi#L474) for Node.js 12.x.